lib/datadog/grok/src/filters/keyvalue.rs

```diff
@@ -301,13 +307,30 @@ fn quoted<'a>(
     }
 }
 
+fn re_find<'a, E>(re: &'a Regex) -> impl Fn(&'a str) -> IResult<&'a str, &'a str, E>
```

Comment:
Huh, yeah, I do see that `nom-regex` takes ownership of the `Regex` but seemingly unnecessarily? I'd open an issue but the source doesn't seem to be published somewhere that would be possible.

Reply:
I think the source is here? https://github.com/rust-bakery/nom-regex

Considering how little we actually used of nom-regex I'm glad to see a dependency dropped from the project.

Reply:
Aha, there it is. And agreed, we were barely using it.